### PR TITLE
Fix for CLOUD-3607, Move Image Builder non maven repo content out of tmp

### PR DESCRIPTION
**Pull request by jfdenise**
Issue: https://issues.redhat.com/browse/CLOUD-3607

### CODE REVIEW

File: jboss/container/wildfly/galleon/build-feature-pack/configure.sh [hunk 1]
```diff
@@ -34,7 +34,9 @@ if [ -f "${ZIPPED_REPO}" ]; then
   unzip ${ZIPPED_REPO} -d /tmp
   repoDir=$(find /tmp -type d -iname "*-image-builder-maven-repository")
   mv $repoDir/maven-repository "$TMP_GALLEON_LOCAL_MAVEN_REPO"
```
Comment (jmesnil) [inline jboss/container/wildfly/galleon/build-feature-pack/configure.sh:36]: I read that as move everything in `$repoDir` other than `maven-repository` to `$JBOSS_CONTAINER_WILDFLY_S2I_GALLEON_DIR/maven-repo-misc`.

Is this only the licenses or other other things moved?

Comment (jfdenise) [reply jboss/container/wildfly/galleon/build-feature-pack/configure.sh:36]: We could have some more, eg: the patches applied, JBossEULA.txt file.

Comment (jmesnil) [reply jboss/container/wildfly/galleon/build-feature-pack/configure.sh:36]: I'm fine with the PR but the title and commit message should reflect that this deals with any extra stuff (not only licenses)

**Comment by jfdenise** [comment]
@jmesnil renamed all.

**Pull request merged**